src/ClientHandle.h

```diff
@@ -244,7 +244,16 @@ class cClientHandle  // tolua_export
 	AString GetLocale(void) const { return m_Locale; }
 
 	int GetUniqueID(void) const { return m_UniqueID; }
+	// tolua_end
+	
+	/// Adds a channel to the handle's collection of registerd channels
+	void RegisterChannel(const AString & a_PluginChannel);
+	
+	/// Removes a channel to the handle's collection of registerd channels
+	void RemoveChannel(const AString & a_PluginChannel);
+	// tolua_begin
```

Comment:
Add an extra empty line above the tolua directive (this way it looks as if grouped to `RemoveChannel`)